test/autorest/generated/custombaseurlgroup/complex.go

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package custombaseurlgroup
+
+import (
+	"context"
+	azinternal "generatortests/autorest/generated/custombaseurlgroup/internal/custombaseurlgroup"
+	"net/url"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+)
+
+// DefaultEndpoint is the default endpoint used for the CustomBaseURLClient service.
+const DefaultEndpoint = "http://localhost:3000"
```

Comment:
This should be omitted.  The custom-baseUrl swagger uses the `x-ms-parameterized-host` extension which means the swagger doesn't define a base URL and expects the caller to provide it.  I realize this doesn't completely map with what we're doing in track 2 since we always require this parameter now.
This got me thinking though, that for swaggers that define a host we should emit a comment in the constructor that says "Use DefaultEndpoint for blah blah" or something.

Reply:
Ah, you're right. Yes that makes sense to me. I removed it.